src/components/commons/input-text-box/cv-input-text-box.tsx

```diff
@@ -38,6 +43,24 @@ const CvInputTextBox: React.FC<CvInputTextBoxProps> = (props: CvInputTextBoxProp
     setInputValue(date);
   };
 
+  const fileChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    e.target.files && props.onChange(e.target.files[0]);
+  };
+
+  const handleToggle = () => {
+    toggleMenu((prevState) => !prevState);
+  };
+
+  const onSelectMenu = (menuItem: IMenuItems) => {
+    props.onChange(menuItem);
+    handleToggle();
+  };
+
+  const onSelectBoxInputChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    props.onChange(e.target.value);
+    setInputValue(e.target.value);
+  };
```

Comment:
what if we need to uplift the selected option object
```js
{
id: ...,
value: ...,
label: ...,
}
```
i think  here we are uplifting only selected value. Current me if i am wrong